source/stable-release_iot/stable-release_iot.rst

```diff
@@ -0,0 +1,17 @@
+.. _stable-celadon-iot:
+
+Stable Celadon IoT
+##################
+
+Celadon IoT is an Internet of Things (IoT) overlay on top of Celadon. Celadon IoT, release provides users with an extensive set of features on Comet Lake(CML), Tiger Lake(TGL), and Elkhart Lake(EHL) Intel® platforms with single Baseline/Image.
+
+Celadon IoT offers 100% compliance for Compatibility Test Suite (CTS), Vendor Test Suite (VTS), CTS-on-GSI, GMS Test Suite (GTS), and Security Test Suite (STS) on Comet Lake(CML), Tiger Lake(TGL), and Elkhart Lake(EHL). 
+
+Celadon IoT empowers the community to utilize 100%compliance on the supported platforms with continuous support for new features and fixes.
```

Comment:
"continuous support for new features and fixes" -> I would just say "continuous updated with ASBs", are you intending to develop new features on this Stable Releases(IoT) branch while in maintenance phase?

Reply:
Updated as per the Suggestion